Makefile

```diff
@@ -1,6 +1,16 @@
-ROLE_LIST := ./ci_framework/roles/*
+SHELL := /bin/bash
+ROLE_LIST ?= ./ci_framework/roles/*
 USE_VENV ?= ${USE_VENV:-yes}
-BUILD_VENV_CTX := yes
+BUILD_VENV_CTX ?= yes
+MOLECULE_CONFIG ?= ${MOLECULE_CONFIG:-.config/molecule/config_podman.yml}
+
+# target vars for generic operator install info 1: target name , 2: operator name
+define vars
+${1}: export ROLE_LIST=${ROLE_LIST}
+${1}: export USE_VENV=${USE_VENV}
+${1}: export BUILD_VENV_CTX=${BUILD_VENV_CTX}
+${1}: export MOLECULE_CONFIG=${MOLECULE_CONFIG}
+endef
```

Comment:
it is not called anywhere can we remove it?

Reply:
the ROLE_LIST is consumed in target, and the USE_VENV + MOLECULE_CONFIG are used in scripts. As I understand the "export" thing here, it's supposed to expose the make "parameters" as environment variable that may be then consumed by sub-shells, such as scripts. So this is what I need.

Reply:
yup, correct, thanks!